::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sun, 6 Aug 2017 20:42:47 +0200
Subject: [PATCH] classes: baremetal: AArch64: virtio NIC names sync

grep -e "ens[[:digit:]]" -R . -l | \
  xargs sed -i \
    -e 's/ens3/enp1s0/g' \
    -e 's/ens4/enp2s0/g' \
    -e 's/ens5/enp3s0/g' \
    -e 's/ens6/enp4s0/g'

Since AArch64 will be using virtio-net-pci NIC model for guests,
predictable interface naming yields a slightly different scheme.

Update all configuration to reflect this.

NOTE: Above configuration is expected with libvirt 3.x, which puts
each NIC on a separate PCIe bus (which also imposes virtio modern
is used).

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 .../baremetal-mcp-ocata-odl-ha/infra/config.yml    |  4 ++--
 .../baremetal-mcp-ocata-odl-ha/infra/kvm.yml       | 18 +++++++++-----
 .../baremetal-mcp-ocata-odl-ha/infra/maas.yml      |  8 +++----
 .../opendaylight/control.yml                       |  2 +-
 .../openstack/compute.yml                          | 28 ++++++++--------------
 .../openstack/control.yml                          |  6 ++---
 .../openstack/dashboard.yml                        |  4 ++--
 .../openstack/database.yml                         |  6 ++---
 .../openstack/message_queue.yml                    |  6 ++---
 .../baremetal-mcp-ocata-odl-ha/openstack/proxy.yml |  6 ++---
 .../openstack/telemetry.yml                        |  6 ++---
 .../baremetal-mcp-ocata-ovs-ha/infra/config.yml    |  4 ++--
 .../baremetal-mcp-ocata-ovs-ha/infra/kvm.yml       | 18 +++++++++-----
 .../baremetal-mcp-ocata-ovs-ha/infra/maas.yml      |  8 +++----
 .../openstack/compute.yml                          | 27 ++++++++-------------
 .../openstack/control.yml                          |  6 ++---
 .../openstack/dashboard.yml                        |  4 ++--
 .../openstack/database.yml                         |  6 ++---
 .../openstack/message_queue.yml                    |  6 ++---
 .../baremetal-mcp-ocata-ovs-ha/openstack/proxy.yml |  6 ++---
 .../openstack/telemetry.yml                        |  6 ++---
 21 files changed, 91 insertions(+), 94 deletions(-)

diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/config.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/config.yml
index a7b08f8..b9bd3e0 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/config.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/config.yml
@@ -33,8 +33,8 @@ parameters:
   linux:
     network:
       interface:
-        ens3: ${_param:linux_dhcp_interface}
-        ens4: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
   salt:
     master:
       accept_policy: open_mode
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/kvm.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/kvm.yml
index f73d48e..32fd829 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/kvm.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/kvm.yml
@@ -25,7 +25,8 @@ parameters:
     cluster_node03_address: ${_param:infra_kvm_node03_address}
     keepalived_vip_interface: br-ctl
     keepalived_vip_virtual_router_id: 69
-    deploy_nic: enp6s0
+    deploy_nic: eth0
+    trunk_nic: eth1
   salt:
     control:
       size: #RAM 4096,8192,16384,32768,65536
@@ -123,7 +124,7 @@ parameters:
   linux:
     network:
       interface:
-        eth3:
+        eth0:
           enabled: true
           type: eth
           proto: manual
@@ -131,6 +132,11 @@ parameters:
           netmask: 255.255.255.0
           name: ${_param:deploy_nic}
           noifupdown: true
+        eth1:
+          enabled: true
+          type: eth
+          proto: manual
+          name: ${_param:trunk_nic}
         br-mgmt:
           enabled: true
           proto: dhcp
@@ -141,13 +147,13 @@ parameters:
           use_interfaces:
           - ${_param:deploy_nic}
           noifupdown: true
-        vlan300:
+        vlan2183:
           enabled: true
           proto: manual
           type: vlan
-          name: ${_param:deploy_nic}.300
+          name: ${_param:trunk_nic}.2183
           use_interfaces:
-          - ${_param:deploy_nic}
+          - ${_param:trunk_nic}
         br-ctl:
           enabled: true
           type: bridge
@@ -155,4 +161,4 @@ parameters:
           address: ${_param:single_address}
           netmask: 255.255.255.0
           use_interfaces:
-          - ${_param:deploy_nic}.300
+          - ${_param:trunk_nic}.2183
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/maas.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/maas.yml
index c4bc929..5649990 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/maas.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/infra/maas.yml
@@ -4,10 +4,10 @@ classes:
 - cluster.baremetal-mcp-ocata-odl-ha.infra
 parameters:
   _param:
-    dhcp_interface: ens3
-    primary_interface: ens4
-    pxe_interface: ens5
-    external_interface: ens6
+    dhcp_interface: enp1s0
+    primary_interface: enp2s0
+    pxe_interface: enp3s0
+    external_interface: enp4s0
     interface_mtu: 1500
     # MaaS has issues using MTU > 1500 for PXE interface
     pxe_interface_mtu: 1500
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/opendaylight/control.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/opendaylight/control.yml
index 0a01af7..367343a 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/opendaylight/control.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/opendaylight/control.yml
@@ -10,7 +10,7 @@ parameters:
   linux:
     network:
       interface:
-        ens3:
+        enp2s0:
           enabled: true
           type: eth
           proto: static
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/compute.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/compute.yml
index 6f0c52c..cd1456a 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/compute.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/compute.yml
@@ -20,10 +20,9 @@ parameters:
     cluster_node03_hostname: ctl03
     cluster_node03_address: ${_param:openstack_control_node03_address}
     nova_vncproxy_url: https://${_param:cluster_public_host}:6080
-    mgmt_nic: enp6s0
-    tenant_nic: enp7s0
+    mgmt_nic: eth0
+    tenant_nic: eth1
     linux_system_codename: xenial
-    external_nic: enp8s0
     interface_mtu: 1500
     keepalived_vip_interface: br-ctl
     keepalived_vip_virtual_router_id: 69
@@ -57,12 +56,6 @@ parameters:
           type: eth
           proto: manual
           name: ${_param:tenant_nic}
-        external_nic:
-          enabled: true
-          name: ${_param:external_nic}
-          mtu: ${_param:interface_mtu}
-          proto: manual
-          type: eth
         br-mesh:
           enabled: true
           type: bridge
@@ -70,19 +63,19 @@ parameters:
           netmask: 255.255.255.0
           mtu: 1500
           use_interfaces:
-          - ${_param:tenant_nic}.302
-        vlan300:
+          - ${_param:tenant_nic}.2185
+        vlan2183:
           enabled: true
           proto: manual
           type: vlan
-          name: ${_param:mgmt_nic}.300
+          name: ${_param:tenant_nic}.2183
           use_interfaces:
-          - ${_param:mgmt_nic}
-        vlan302:
+          - ${_param:tenant_nic}
+        vlan2185:
           enabled: true
           proto: manual
           type: vlan
-          name: ${_param:tenant_nic}.302
+          name: ${_param:tenant_nic}.2185
           use_interfaces:
           - ${_param:tenant_nic}
         br-ctl:
@@ -92,7 +85,7 @@ parameters:
           address: ${_param:single_address}
           netmask: 255.255.255.0
           use_interfaces:
-          - ${_param:mgmt_nic}.300
+          - ${_param:tenant_nic}.2183
         br-floating:
           enabled: true
           type: ovs_bridge
@@ -109,7 +102,6 @@ parameters:
           address: ${_param:external_address}
           netmask: 255.255.255.0
           use_interfaces:
-          - ${_param:external_nic}
+          - ${_param:tenant_nic}
           use_ovs_ports:
           - float-to-ex
-
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/control.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/control.yml
index e8666d6..149d29d 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/control.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/control.yml
@@ -23,7 +23,7 @@ classes:
 - cluster.baremetal-mcp-ocata-odl-ha.infra
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 50
     cluster_vip_address: ${_param:openstack_control_address}
     cluster_local_address: ${_param:single_address}
@@ -37,8 +37,8 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
   keystone:
     server:
       cacert: /etc/ssl/certs/mcp_os_cacert
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/dashboard.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/dashboard.yml
index 9748c4c..a5317b1 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/dashboard.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/dashboard.yml
@@ -7,5 +7,5 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/database.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/database.yml
index 6c26d26..d74e5ad 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/database.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/database.yml
@@ -16,7 +16,7 @@ classes:
 - cluster.baremetal-mcp-ocata-odl-ha
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 80
     galera_server_cluster_name: openstack_cluster
     cluster_vip_address: ${_param:openstack_database_address}
@@ -30,5 +30,5 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/message_queue.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/message_queue.yml
index f5312c1..d2e6a40 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/message_queue.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/message_queue.yml
@@ -7,7 +7,7 @@ classes:
 - cluster.baremetal-mcp-ocata-odl-ha
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 90
     cluster_vip_address: ${_param:openstack_message_queue_address}
     cluster_local_address: ${_param:single_address}
@@ -20,5 +20,5 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/proxy.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/proxy.yml
index 93abf2a..8a62cee 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/proxy.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/proxy.yml
@@ -15,7 +15,7 @@ classes:
 # - cluster.baremetal-mcp-ocata-odl-ha.stacklight.proxy
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 240
     nginx_proxy_ssl:
       enabled: true
@@ -27,8 +27,8 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
     system:
       package:
         libapache2-mod-wsgi:
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/telemetry.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/telemetry.yml
index 4a56b65..128ec1b 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/telemetry.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-odl-ha/openstack/telemetry.yml
@@ -10,7 +10,7 @@ classes:
 - cluster.baremetal-mcp-ocata-odl-ha.infra
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 230
     cluster_vip_address: ${_param:openstack_telemetry_address}
     cluster_local_address: ${_param:single_address}
@@ -23,5 +23,5 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/config.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/config.yml
index a75b41d..3979fd1 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/config.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/config.yml
@@ -32,8 +32,8 @@ parameters:
   linux:
     network:
       interface:
-        ens3: ${_param:linux_dhcp_interface}
-        ens4: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
   salt:
     master:
       accept_policy: open_mode
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/kvm.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/kvm.yml
index d919fe7..5025cd4 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/kvm.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/kvm.yml
@@ -25,7 +25,8 @@ parameters:
     cluster_node03_address: ${_param:infra_kvm_node03_address}
     keepalived_vip_interface: br-ctl
     keepalived_vip_virtual_router_id: 69
-    deploy_nic: enp6s0
+    deploy_nic: eth0
+    trunk_nic: eth1
   salt:
     control:
       size: #RAM 4096,8192,16384,32768,65536
@@ -115,7 +116,7 @@ parameters:
   linux:
     network:
       interface:
-        eth3:
+        eth0:
           enabled: true
           type: eth
           proto: manual
@@ -123,6 +124,11 @@ parameters:
           netmask: 255.255.255.0
           name: ${_param:deploy_nic}
           noifupdown: true
+        eth1:
+          enabled: true
+          type: eth
+          proto: manual
+          name: ${_param:trunk_nic}
         br-mgmt:
           enabled: true
           proto: dhcp
@@ -133,13 +139,13 @@ parameters:
           use_interfaces:
           - ${_param:deploy_nic}
           noifupdown: true
-        vlan300:
+        vlan2183:
           enabled: true
           proto: manual
           type: vlan
-          name: ${_param:deploy_nic}.300
+          name: ${_param:trunk_nic}.2183
           use_interfaces:
-          - ${_param:deploy_nic}
+          - ${_param:trunk_nic}
         br-ctl:
           enabled: true
           type: bridge
@@ -147,4 +153,4 @@ parameters:
           address: ${_param:single_address}
           netmask: 255.255.255.0
           use_interfaces:
-          - ${_param:deploy_nic}.300
+          - ${_param:trunk_nic}.2183
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/maas.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/maas.yml
index d193469..73215ee 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/maas.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/infra/maas.yml
@@ -4,10 +4,10 @@ classes:
 - cluster.baremetal-mcp-ocata-ovs-ha.infra
 parameters:
   _param:
-    dhcp_interface: ens3
-    primary_interface: ens4
-    pxe_interface: ens5
-    external_interface: ens6
+    dhcp_interface: enp1s0
+    primary_interface: enp2s0
+    pxe_interface: enp3s0
+    external_interface: enp4s0
     interface_mtu: 1500
     # MaaS has issues using MTU > 1500 for PXE interface
     pxe_interface_mtu: 1500
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/compute.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/compute.yml
index 070ab78..88150b0 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/compute.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/compute.yml
@@ -19,9 +19,8 @@ parameters:
     cluster_node03_hostname: ctl03
     cluster_node03_address: ${_param:openstack_control_node03_address}
     nova_vncproxy_url: https://${_param:cluster_public_host}:6080
-    mgmt_nic: enp6s0
-    tenant_nic: enp7s0
-    external_nic: enp8s0
+    mgmt_nic: eth0
+    tenant_nic: eth1
     linux_system_codename: xenial
     interface_mtu: 1500
     keepalived_vip_interface: br-ctl
@@ -40,12 +39,6 @@ parameters:
           type: eth
           proto: manual
           name: ${_param:tenant_nic}
-        external_nic:
-          enabled: true
-          name: ${_param:external_nic}
-          mtu: ${_param:interface_mtu}
-          proto: manual
-          type: eth
         br-mesh:
           enabled: true
           type: bridge
@@ -53,19 +46,19 @@ parameters:
           netmask: 255.255.255.0
           mtu: 1500
           use_interfaces:
-          - ${_param:tenant_nic}.302
-        vlan300:
+          - ${_param:tenant_nic}.2185
+        vlan2183:
           enabled: true
           proto: manual
           type: vlan
-          name: ${_param:mgmt_nic}.300
+          name: ${_param:tenant_nic}.2183
           use_interfaces:
-          - ${_param:mgmt_nic}
-        vlan302:
+          - ${_param:tenant_nic}
+        vlan2185:
           enabled: true
           proto: manual
           type: vlan
-          name: ${_param:tenant_nic}.302
+          name: ${_param:tenant_nic}.2185
           use_interfaces:
           - ${_param:tenant_nic}
         br-ctl:
@@ -75,7 +68,7 @@ parameters:
           address: ${_param:single_address}
           netmask: 255.255.255.0
           use_interfaces:
-          - ${_param:mgmt_nic}.300
+          - ${_param:tenant_nic}.2183
         br-floating:
           enabled: true
           type: ovs_bridge
@@ -92,6 +85,6 @@ parameters:
           address: ${_param:external_address}
           netmask: 255.255.255.0
           use_interfaces:
-          - ${_param:external_nic}
+          - ${_param:tenant_nic}
           use_ovs_ports:
           - float-to-ex
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/control.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/control.yml
index a5913dd..638abba 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/control.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/control.yml
@@ -23,7 +23,7 @@ classes:
 - cluster.baremetal-mcp-ocata-ovs-ha.infra
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 50
     cluster_vip_address: ${_param:openstack_control_address}
     cluster_local_address: ${_param:single_address}
@@ -37,8 +37,8 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
   keystone:
     server:
       cacert: /etc/ssl/certs/mcp_os_cacert
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/dashboard.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/dashboard.yml
index 39cc073..18f622c 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/dashboard.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/dashboard.yml
@@ -7,5 +7,5 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/database.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/database.yml
index dabda49..7a6934b 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/database.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/database.yml
@@ -16,7 +16,7 @@ classes:
 - cluster.baremetal-mcp-ocata-ovs-ha
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 80
     galera_server_cluster_name: openstack_cluster
     cluster_vip_address: ${_param:openstack_database_address}
@@ -30,5 +30,5 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/message_queue.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/message_queue.yml
index 881644b..f510a7b 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/message_queue.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/message_queue.yml
@@ -7,7 +7,7 @@ classes:
 - cluster.baremetal-mcp-ocata-ovs-ha
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 90
     cluster_vip_address: ${_param:openstack_message_queue_address}
     cluster_local_address: ${_param:single_address}
@@ -20,5 +20,5 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/proxy.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/proxy.yml
index 2147741..4e25b78 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/proxy.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/proxy.yml
@@ -15,7 +15,7 @@ classes:
 # - cluster.baremetal-mcp-ocata-ovs-ha.stacklight.proxy
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 240
     nginx_proxy_ssl:
       enabled: true
@@ -27,8 +27,8 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
     system:
       package:
         libapache2-mod-wsgi:
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/telemetry.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/telemetry.yml
index 505ee7f..6ad13ae 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/telemetry.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-ovs-ha/openstack/telemetry.yml
@@ -10,7 +10,7 @@ classes:
 - cluster.baremetal-mcp-ocata-ovs-ha.infra
 parameters:
   _param:
-    keepalived_vip_interface: ens3
+    keepalived_vip_interface: enp2s0
     keepalived_vip_virtual_router_id: 230
     cluster_vip_address: ${_param:openstack_telemetry_address}
     cluster_local_address: ${_param:single_address}
@@ -23,5 +23,5 @@ parameters:
   linux:
     network:
       interface:
-        ens2: ${_param:linux_dhcp_interface}
-        ens3: ${_param:linux_single_interface}
+        enp1s0: ${_param:linux_dhcp_interface}
+        enp2s0: ${_param:linux_single_interface}
